src/cmd/flux-mini.py

```diff
@@ -903,13 +917,14 @@ def event_watch_cb(self, future, args, jobinfo, label=""):
             )
         elif event.name == "alloc":
             jobinfo["state"] = "running"
-        elif event.name == "start" and args.watch:
+        elif event.name == "start" and (args.watch or args.wait.startswith("exec.")):
             #
             #  Watch the exec eventlog if the --watch option was provided:
```

Comment:
minor comment typo: probably should colon end of line since the sentence continues now